src/main.js

```diff
@@ -16,17 +16,32 @@ const checkValidationOfValues = envSpecString => {
     "week"
   ];
   const alphanumericThatDoesNotStartWithDigit = /^[A-Z_][0-9A-Z_]*$/;
+  const genericForCheckingRestrChoicesSyntax = /^\[(.*)\]$/;
   let envValArray = parseVarFromType(envSpecString.trim().split("\n")); //split lines based on \n character and parse them
   let checkValidation = true;
   //envValArray is a two-dimensional array containing only the types and variables
   //element[0] is the variable e.g. ADMIN_EMAIL
   //element[1] is the type e.g. test
-  envValArray = envValArray.filter(element => {
-    //check for valid variables AND types
+  //element[2] will be what this environmental variable contains e.g. as default or as options
+  envValArray = envValArray.map(element => {
```

Comment:
OK, we will have to make this human-friendlier by making a couple of changes:

1. Use a **new** _constant_ to store the result of `map` (e.g. `envSpecEntries`)
2. Use a structured object, instead of an array:
    ```javascript
    // Prefer this
    const entry = {
        name: 'ADMIN_EMAIL',
        type: 'test',  // If restricted choices are set, this should be `null`
        choices: null  // If restricted choices are set, this should be an array (e.g. `['a', 'b', 'c']`)
    }

    // Instead of this
    const entryInArrayFormat = ['ADMIN_EMAIL', 'email', null];

    console.log(entry.name);  // It's clear what we are printing here
    console.log(entry[0]);  // It's not clear what we are printing here
    ```